app/views/solidus_friendly_promotions/admin/promotions/_table.html.erb

```diff
@@ -25,7 +25,7 @@
           </td>
           <td>
             <% if promotion.usage_limit.nil? %>
-              ∞
+              <%= promotion.discounted_orders.exists? ? t(:say_yes, scope: :spree) : t(:say_no, scope: :spree) %>
```

Comment:
I do not understand what a "yes" or "no" means in context of a `nil` usage_count? Shouldn't this show the count of discounted orders instead?

Reply:
`usage_count` is never `nil` - it can be zero for unused orders, or some really large number for used ones.

Reply:
So rather than tell the user nothing, we can tell them whether their promotion has been used in any complete order.

Reply:
why not display discounted orders count?